lib/drives/models/drive.dart

```diff
@@ -72,17 +73,14 @@ class Drive extends Trip {
     };
   }
 
-  List<Ride>? get approvedRides => rides?.where((ride) => ride.status == RideStatus.approved).toList();
-  List<Ride>? get pendingRides => rides?.where((ride) => ride.status == RideStatus.pending).toList();
-
-  static Future<List<Drive>> getDrivesOfUser(int userId) async {
-    return Drive.fromJsonList(await SupabaseManager.supabaseClient.from('drives').select().eq('driver_id', userId));
-  }
+  List<Ride>? get approvedRides => rides?.where((Ride ride) => ride.status == RideStatus.approved).toList();
+  List<Ride>? get pendingRides => rides?.where((Ride ride) => ride.status == RideStatus.pending).toList();
 
   static Future<bool> userHasDriveAtTimeRange(DateTimeRange range, int userId) async {
-    //get all upcoming drives of user
-    List<Drive> drives = await Drive.getDrivesOfUser(userId);
-    drives = drives.where((drive) => !drive.cancelled && !drive.isFinished).toList();
+    List<Map<String, dynamic>> data =
```

Comment:
Beim Drive erstellen:
```
E/flutter ( 4364): [ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: type 'List<dynamic>' is not a subtype of type 'List<Map<String, dynamic>>'
E/flutter ( 4364): #0      Drive.userHasDriveAtTimeRange (package:motis_mitfahr_app/drives/models/drive.dart:80:32)
E/flutter ( 4364): <asynchronous suspension>
E/flutter ( 4364): #1      _CreateDriveFormState._onSubmit (package:motis_mitfahr_app/drives/pages/create_drive_page.dart:108:13)
E/flutter ( 4364): <asynchronous suspension>
E/flutter ( 4364):
```

Reply:
Thanks! I have now searched by hand for all occurences of `List<Map<String, dynamic>>`, I should now have actually found all of them.